### PR TITLE
Improve simulation failsafe doc

### PR DESCRIPTION
**Pull request by hamishwillee**
Follows on from #729

### CODE REVIEW

File: en/simulation/failsafes.md [hunk 1]
```diff
@@ -1,26 +1,49 @@
...
 
-To control how fast the battery depletes to this value use the parameter `SIM_BAT_DRAIN`. By changing this configuration in flight you can also test regaining capacity to simulate inaccurate battery state estimation or in air charging technology.
+The simulated battery is implemented to never run out of energy, and by default only depletes to 50% of its capacity. 
+This enables testing of battery indication in GCS UIs without triggering low battery reactions that might interrupt other testing.
+
+To change this minimal battery percentage value change [this line](https://github.com/PX4/Firmware/blob/9d67bbc328553bbd0891ffb8e73b8112bca33fcc/src/modules/simulator/simulator_mavlink.cpp#L330).
+
+To control how fast the battery depletes to the minimal value use the parameter [SIM_BAT_DRAIN](../advanced/parameter_reference.md#SIM_BAT_DRAIN).
+
+> **Tip** By changing this configuration in flight, you can also test regaining capacity to simulate inaccurate battery state estimation or in-air charging technology.
```
Comment (hamishwillee) [inline en/simulation/failsafes.md:44]: @MaEtUgR How can you test this in flight? Changing the percentage is a code change, and SIM_BAT_DRAIN does not appear to have a negative value (ie for charging)

**Comment by hamishwillee** [comment]
@MaEtUgR This is merged, but would appreciate your review just to confirm I didn't change the "meaning" of your work. 

Awesome job on this BTW. Really useful doc.

File: en/simulation/failsafes.md (hunk 1 above)
Comment (MaEtUgR) [reply en/simulation/failsafes.md:44]: `SIM_BAT_DRAIN` is the time in seconds until the battery empties to the value you specified. If it emptied let's say in 60s and you increase the value to 1000s the battery voltage will rise again 🔋
I should also make a simulation only parameter out of the other setting sucht that recompilation is not needed.

**Comment by MaEtUgR** [comment]
Thanks for taking over! I need to go over it again. Not because of your changes but because I found out it's not all exactly as I wrote. Also I wrote:

> Note that the under link loss documented behavior is the one prepared in https://github.com/PX4/Firmware/pull/11357

and the pr is not yet merged because of a debate... 😛

**Comment by hamishwillee** [comment]
@MaEtUgR Thanks for letting me know. Sorry about premature merging. Should we remove, or are you going to get this sorted out in the coming weeks?

**Comment by MaEtUgR** [comment]
@hamishwillee You did just fine, I did the mistakes here, corrected in #736

**Pull request merged**